cerberus-web/src/test/java/com/nike/cerberus/util/AuthTokenGeneratorTest.java

```diff
@@ -0,0 +1,31 @@
+package com.nike.cerberus.util;
+
+import java.security.NoSuchAlgorithmException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AuthTokenGeneratorTest {
+
+  @Test
+  public void testAuthTokenGeneratorIfLengthIsConfiguredWrongly() {
+    String exceptionMessage = "";
+    Exception exception = null;
+    try {
+      new AuthTokenGenerator(0);
+    } catch (NoSuchAlgorithmException | IllegalArgumentException e) {
+      exceptionMessage = e.getMessage();
+      exception = e;
+    }
+    Assert.assertNotNull(exception);
+    Assert.assertEquals(
+        "${cerberus.auth.token.generate.length} must be at least 64 but was 0", exceptionMessage);
+  }
+
+  @Test
+  public void testAuthTokenGenerateShouldGenerateNewToken() throws NoSuchAlgorithmException {
```

Comment:
Why this test throw an exception when it should be successful?

Reply:
So, it is a compile time exception where we can have try catch or add throws exception

Reply:
Catching it seems like a better idea than throwing it here

Reply:
ok, made changes as per comments